recipe/meta.yaml

```diff
@@ -15,16 +15,19 @@ source:
     - dlopen_test.patch  # [osx]
 
 build:
-  number: 0
-  skip: True  # [win]  
+  number: 1
+  skip: True  # [win]
 
 requirements:
   build:
     - m4
-    - gmp 6.1.*
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - gmp
     - toolchain3
```

Comment:
`toolchain3` should have been removed as well. Handling in PR ( https://github.com/conda-forge/nettle-feedstock/pull/3 ).

cc @CJ-Wright (for awareness)